xla/service/gpu/runtime/gpublas_lt_matmul.cc

```diff
@@ -109,6 +109,11 @@ absl::Status DoMatmul(
   }));
 
   TF_ASSIGN_OR_RETURN(auto algos, (*plan)->GetAlgorithms());
+  if((size_t)algorithm >= algos.size()) {
+    return absl::InternalError("A required gpublas-lt matmul algorithm is not found. "
```

Comment:
`absl::StrCat` or `abst::StrFormat` instead of string interpolation and std::to_string